lib/active_fedora/orders/association.rb

```diff
@@ -82,6 +82,12 @@ def delete_at(loc)
       target.delete_at(loc)
     end
 
+    # Delete all occurences of the specified target
+    # @param obj object to delete
+    def delete(obj)
```

Comment:
@tpendragon Okay, so we can still have `Orders::TargetProxy#delete` so long as we name the method on  `Orders::Association` to `#delete_target`?